### PR TITLE
wip: document the new (upcoming) slack channels for the Falco projects on the Kubernetes Slack

### PR DESCRIPTION
**Pull request by leodido**
Signed-off-by: Leonardo Di Donato <leodidonato@gmail.com>

### CODE REVIEW

File: slack/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    - falcoctl
+    - helm chart
+    - security hub
+    - exporter
+    - sidekick
+    - other sub-projects
+- #falco-maintainers
+    - public
+    - AMA
+- #falco-maintainers-private
```
Comment (leodido) [inline slack/README.md:26]:
```suggestion
- #falco-maintainers-pvt
```

**Comment by leodido** [comment]
Request for new channels sent: https://github.com/kubernetes/community/pull/4792

**Comment by leogr** [comment]
Question:
Would we like to add direct links to channels (eg. https://kubernetes.slack.com/archives/CMWH3EH32 ) ?

**Comment by leodido** [comment]
Yep, once we have channels. :)

This is an on-going effort to document the new Slack

**Comment by leodido** [comment]
Closing this since PR https://github.com/kubernetes/community/pull/4792 has been closed

/close

**Pull request closed without merging**